### PR TITLE
Pc doc

### PR DESCRIPTION
**Pull request by Faiz**
## What does this PR do?
Adds docs for pc availability.

### CODE REVIEW

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/97/
Jenkins statics successfully built.

File: frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  shell: `curl https://uclapi.com/resources/desktop_availability \
+-d token=uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb
+`,
+
+  javascript: `fetch("https://uclapi.com/resources/desktop_availability?token=uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb")
+.then((response) => {
+  return response.json()
+})
+.then((json) => {
+  console.log(json);
```
Comment (ChristopherHammond13) [inline frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx]: Is this `console.log` needed?

File: frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+import Cell from './../../Cell.jsx';
+
+
+let codeExamples = {
+  python: `import requests
+
+params = {
+  "token": "uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb",
+}
+r = requests.get("https://uclapi.com/resources/desktop_availability", params=params)
```
Comment (ChristopherHammond13) [inline frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx]: isn't the endpoint `resources/desktops`?

File: frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            </Table>
+          </Topic>
+
+          <Topic
+            noExamples={true}>
+            <Table
+              name="Errors">
+              <Cell
+                name="Could not parse the desktop availability data. Please try again later or contact us for support"
+                description="We don't currently have the upto date data. Wait a while before you make next request." />
```
Comment (ChristopherHammond13) [inline frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx]: `up to date`

File: frontend/src/react/components/documentation/Routes/Resources/VersionHeader.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      return (
+        <Topic
+          noExamples={true}>
+          <p>
+            The base url is <code>https://uclapi.com/resources/</code>
+          </p>
+          <Table>
+            <Cell
+              name="Version Header"
+              example="uclapi-roombookings-version" />
```
Comment (ChristopherHammond13) [inline frontend/src/react/components/documentation/Routes/Resources/VersionHeader.jsx]: Shouldn't this be `uclapi-resources-version`?

File: frontend/src/react/components/documentation/Sidebar.jsx [hunk 5]
```diff
@@ -145,6 +145,18 @@ export default class Sidebar extends React.Component {
...
           />
+          
+          <ListItem
+            href="#resources"
+            primaryText="Resources"
+            primaryTogglesNestedList={true}
+            nestedItems={[
+              <ListItem
+                primaryText="Get Desktop availability"
+                href="#resources/desktop_availability"
```
Comment (ChristopherHammond13) [inline frontend/src/react/components/documentation/Sidebar.jsx]: again, `resources/desktops`

**Comment by Faiz** [comment]
done

**Comment by ChristopherHammond13** [comment]
Build finished. 
Jenkins statics successfully built.

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/118/
Jenkins statics successfully built.

File: backend/uclapi/resources/views.py [hunk 6]
```diff
@@ -39,7 +39,7 @@ def get_pc_availability(request, *args, **kwargs):
         _ = pc.get
         data.append({
             "location": {
-                "room_name": _("location"),
+                "roomname": _("location"),
```
Comment (ChristopherHammond13) [inline backend/uclapi/resources/views.py]: Any reason for this change?

File: frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx (hunk 1 above)
Comment (ChristopherHammond13) [reply frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx]: bump

File: backend/uclapi/resources/views.py (hunk 6 above)
Comment (Faiz) [reply backend/uclapi/resources/views.py]: yep! cos we use roomname in other parts of the api

File: frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx (hunk 1 above)
Comment (Faiz) [reply frontend/src/react/components/documentation/Routes/Resources/DesktopAvailability.jsx]: ye! how else do we print it to the screen? we do the same wiht python and shell

**Comment by Faiz** [comment]
yo merge it @ChristopherHammond13 :D

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/214/
Jenkins statics successfully built.

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/215/
Jenkins statics successfully built.

**Pull request merged**